mpsl/CMakeLists.txt

```diff
@@ -19,4 +19,11 @@ zephyr_include_directories(include)
 zephyr_include_directories(include/protocol)
 zephyr_link_libraries(${MPSL_LIB_DIR}/libmpsl.a)
 
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/fem)
+  # This is done like this instead of including a CMakeLists.txt file from the fem directory,
+  # as the whole fem directory is deleted and then the artifacts are copied inside it during MPSL deploy.
+  # That is why the CMake file handling FEMs must be currently kept in the nrfxlib/mpsl directory.
+  include(fem.cmake) # Should be done uncondtionally when the split libraries are deployed to nrfxlib.
+endif()
+
```

Comment:
I'm not sure if the FEM part will be ever really indepenedent from the MPSL library - as for now it is dependent, and we haven't yet performed an analysis if it is possible to make it independent.

I think the main goal was to split the internal FEM and MPSL projects, in order to make introducing changes to the FEM library a quicker process. This does not need making the FEM library independent from the MPSL library.

I personally think that keeping the fem directory inside the mpsl library as long as they are not independent directories is a bit easier for now at requires less maintenence. Moving the libraries to a different directory shouldn't be as big of a problem, as it will mostly require only changes in nrfxlib.

Reply:
Also I do not see a clear way of performing conditional inclusion of the FEM libraries in a similar way it is done now - it helps with a smooth transition, as we could merge this PR without updating the libraries.

Reply:
as discussed f2f, let's leave in mpsl/fem subdir as you propose.